dataactbroker/scripts/update_historical_duns.py

```diff
@@ -0,0 +1,131 @@
+import logging
+import boto
+import os
+import pandas as pd
+import argparse
+from datetime import datetime
+
+from dataactcore.models.domainModels import DUNS
+from dataactcore.scripts.loadDUNS import load_duns_by_row
+from dataactvalidator.scripts.loaderUtils import clean_data
+from dataactvalidator.health_check import create_app
+from dataactcore.interfaces.db import GlobalDB
+from dataactcore.logging import configure_logging
+from dataactcore.config import CONFIG_BROKER
+
+logger = logging.getLogger(__name__)
+
+# CSV column header name in DUNS file
+column_headers = [
+    "awardee_or_recipient_uniqu",  # DUNS Field
+    "registration_date",  # Registration_Date
+    "expiration_date",  # Expiration_Date
+    "last_sam_mod_date",  # Last_Update_Date
+    "activation_date",  # Activation_Date
+    "legal_business_name"  # Legal_Business_Name
+]
+
+
+def remove_existing_duns(data, sess):
+    """Remove rows frome file that alread have a entry in broker database. We should only update missing DUNS"""
```

Comment:
Typo. `Remove rows from file that already have a entry in broker database. We should only update missing DUNS`

Reply:
Updating in next commit.